src/components/home/elements/NavBar.vue

```diff
@@ -130,12 +135,104 @@ export default {
             }
         }
     },
+    data() {
+        return {
+            interval: 0,
+            response: '',
+            currentTime: '',
+            showHeight: false,
+            currentHeight: ''
+        }
+    },
+    created() {
+        this.showHeight = this.getHeightStatus()
+        if (this.showHeight) {
+            this.childByValue(this.showHeight)
+        }
+        this.currentHeight = this.getCurrentHeight()
+        this.currentTime = this.getCurrentTime()
+    },
     mounted() {
         jdenticon()
     },
     computed: {
     },
+    beforeDestroy() {
+        if (this.interval) {
+            clearInterval(this.interval)
+            this.interval = 0
+        }
+    },
     methods: {
+        getTime(date) {
```

Comment:
This method shows UTC time, not local time. 
(I think just use `new Date(timestamp).toLocaleString()` to display the time is OK. No need write this function.)